### ISSUE TITLE
Enable parallel work item uploads to helix sdk

### ISSUE THREAD

**Issue by alexperovich** [dotnet/arcade#1483]
The Helix SDK currently doesn't do work item payload uploads in parallel. Change this to be run in parallel to increase performance for large amounts of payloads.

**Comment by markwilkie** [comment]
@Chrisboh - here's another item which may need to be done sooner than later for ASP.

**Comment by jonfortescue** [comment]
Waiting on updates from ASP.NET before marking as closed.

**Comment by jonfortescue** [comment]
@HaoK tested this change in https://github.com/aspnet/AspNetCore/pull/7373 and it seems to have worked. Closing.

**Comment by HaoK** [comment]
No, we weren't able to upgrade to newest SDKs due to the helix errors (unable to find any jobs), I only tested the workaround for the warning

**Comment by HaoK** [comment]
We are still using a really old sdk build

**Comment by jonfortescue** [comment]
Oh, nvm then! Reopening.

**Comment by HaoK** [comment]
I can open a new PR with just SDK version update

**Comment by jonfortescue** [comment]
@HaoK I think that'd be great. Link me to it and I'll see if I can help sort out the update issues.

**Comment by HaoK** [comment]
What's the best sdk version to try, we are still on 2.0.0-beta.19073.6

**Comment by jonfortescue** [comment]
Probably 2.0.0-beta.19105.2 would be best.

**Comment by HaoK** [comment]
Ok we can use https://github.com/aspnet/AspNetCore/pull/7692 for this

**Comment by HaoK** [comment]
Yeah something changed that we need to react to in the sdk
```
D:\a\1\s\.nuget\packages\microsoft.dotnet.helix.sdk\2.0.0-beta.19105.2\tools\Microsoft.DotNet.Helix.Sdk.MonoQueue.targets(51,5): error : SendHelixJob given no WorkItems to send. [D:\a\1\s\eng\helix\helix.proj]
```

**Comment by jonfortescue** [comment]
Confirmed that this is now working for ASP.NET. Closing.
